backend/migrations/sqls/20211130171433-changeStatusColsInComponentsTable-down.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE komponenten RENAME TO komponenten_alt;
+
+CREATE TABLE komponenten(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR,
+    aktuellerStatus INTEGER,
+    naechsterStatus INTEGER,
```

Comment:
brauchen wir aktueller status und nächster status? im frontend werten wir ja einfach aus zwischen welchen dates das aktuelle date grad ist und setzen dann den entsprechenden componentStatus

Reply:
Das ist die DB migrate down datei. Die ist dafür da um den alten Stand der DB wiederherzustellen. Den neuen Stand der DB siehst du in db migrate up